src/components/balanceDetails.module.css

```diff
@@ -0,0 +1,37 @@
+.container {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	padding: 0.9rem 0;
```

Comment:
this css is globally scoped seems quite generic and could clash later on? is this due to how Gatsby works?

Reply:
From what I understood of the CSS modules, everything should be locally scoped. The only things that have obviously affected other parts of the CSS were when I set some rules for the section element. I haven’t noticed any issues from using .container, but if I notice any wonkiness it’ll be easy enough to change the name to  .[componentName]Container.

Reply:
+1